src/components/DivWithCenterdChildren/index.js

```diff
@@ -32,7 +32,9 @@ export const ReusableDiv = ({
       )}
       style={{
         backgroundImage: `url(${bgImage})`,
-        backgroundSize: 'cover',
+        backgroundRepeat: 'no-repeat',
+        backgroundPosition: 'center top',
+        backgroundSize: '100%',
```

Comment:
those aren't a props, I'm changing the div background image size to fit our uses because make it cover makes me not able to scale it in different screen sizes.